### DISCUSSION TITLE
Replacing displayed row string

### DISCUSSION THREAD

**Discussion by lucasadelino** [davatorium/rofi#1991] [Q&A]
Hello,

I'm trying to integrate rofi -dmenu into one of my scripts. Here's my use case: I have a bunch of names, each associated with an id number, like: ('blob': '341312132'). What I want to do is search for names, and return their ids. Ideally, only the names would be displayed as rows in rofi, and only the id would be returned.

If I'm understanding correctly, I should be able to do this using:

echo -en "341312132\0display\x1fblob\n" | rofi -dmenu

However, that doesn't work: the text that shows up in the row is 341312132, not 'blob'. 

Any ideas for how to do this? I must be missing something really obvious.

**Comment by DaveDavenport** [comment]
That line works for me.  Are you on the right version (git).
